src/test/java/com/jcabi/log/VerboseProcessTest.java

```diff
@@ -310,6 +313,83 @@ public void terminatesMonitorsAndProcessIfClosedNormal() throws Exception {
     }
 
     /**
+     * VerboseProcess can run a java process which will throw a stack trace and
+     * makes sure it will group the stack trace.
+     */
+    @Test
+    public void logCompleteStackTrace() {
+        final org.apache.log4j.Logger logger =
+                org.apache.log4j.Logger.getRootLogger();
+        final VerboseProcessTest.TestAppender appender =
+                new VerboseProcessTest.TestAppender();
+        logger.addAppender(appender);
+        final String[] commands = {
+            retrieveJavaExecLocation(), "-cp",
+            System.getProperty("java.class.path"),
+            "com.jcabi.log.VerboseProcessExample",
+        };
+        final ProcessBuilder builder = new ProcessBuilder(commands);
+        final VerboseProcess process = new VerboseProcess(
+            builder, Level.INFO, Level.SEVERE
+        );
+        boolean failed = false;
+        try {
+            process.stdout();
+        } catch (final IllegalArgumentException ex) {
+            failed = true;
```

Comment:
@dean-e-clark instead of 3 lines around `faled` we could have just one line with `Assert.fail` just after `process.stdout();`